rasa/nlu/persistor.py

```diff
@@ -151,11 +151,11 @@ def _persist_tar(self, file_key: Text, tar_path: Text) -> None:
         with open(tar_path, "rb") as f:
             self.s3.Object(self.bucket_name, file_key).put(Body=f)
 
-    def _retrieve_tar(self, target_filename: Text) -> None:
+    def _retrieve_tar(self, model_full_path: Text) -> None:
+        tar_name = os.path.basename(model_full_path)
```

Comment:
this line should move one down (below the docstring)

Reply:
Done!